test/test_softclip_layer.h

```diff
@@ -0,0 +1,77 @@
+/*
+    Copyright (c) 2019, Dejan Milosavljevic and the respective contributors
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
```

Comment:
I think it's preferrable to use uniformed copyright notice throughout library files.
Can you use below copyright notice?

```cpp
/*
    Copyright (c) 2013, Taiga Nomi and the respective contributors
    All rights reserved.
    Use of this source code is governed by a BSD-style license that can be found
    in the LICENSE file.
*/
```

By the way, you can update `AUTHORS` file with `scripts/update-authors.sh` file.

Reply:
OK, I'll fix that.
Also I'll discard this PL after create new one for this feature.
Appears that I have small bug in JSON string in test_serialization.h.